### PR TITLE
Fixed edge consolidation

### PR DESCRIPTION
**Pull request by philip-paul-mueller**
The main issue was a typo were `src` was accessed instead of `dst`.

### CODE REVIEW

File: dace/sdfg/utils.py [hunk 1]
```diff
@@ -641,9 +644,10 @@ def remove_edge_and_dangling_path(state: SDFGState, edge: MultiConnectorEdge):
         else:
             if e.dst_conn:
                 e.dst.remove_in_connector(e.dst_conn)
-                e.src.remove_out_connector('OUT' + e.dst_conn[2:])
+                e.dst.remove_out_connector('OUT' + e.dst_conn[2:])
 
         # Continue traversing upwards
+        # TODO: What is about the children of the tree.
```
Comment (tbennun) [inline dace/sdfg/utils.py:650]: What does this mean?

Comment (philip-paul-mueller) [reply dace/sdfg/utils.py:650]: It was a question for you (not you personally but the DaCe Developer).
I was just wondering, why the children where not handled?

Comment (philip-paul-mueller) [reply dace/sdfg/utils.py:650]: Okay this makes sense.

Comment (philip-paul-mueller) [reply dace/sdfg/utils.py]: Updated.

**Review by tbennun — APPROVED** [review]

**Pull request merged**